antsibull/cli/antsibull_build.py

```diff
@@ -88,12 +88,16 @@ def _normalize_release_build_options(args: argparse.Namespace) -> None:
     if args.command not in ('single', 'multiple', 'rebuild-single'):
         return
 
+    if args.build_data_dir is None:
+        args.build_data_dir = args.dest_dir
+
     if args.build_file is None:
         args.build_file = (DEFAULT_FILE_BASE
                            + f'-{args.ansible_version.major}.{args.ansible_version.minor}.build')
 
-    if not os.path.isfile(args.build_file):
-        raise InvalidArgumentError(f'The build file, {args.build_file} must already exist.'
```

Comment:
args.build_data_dir probably needs to have expanduser, expandvars run on it at some point, similar to how args.dest_dir is done.

Thinking about cornercases, args.build_file, args.pieces_file, and args.deps_file probably need to have that run on them as well since they could be specified with a non-absolute path.  I'll open a bug about that since it's pre-existing.

I also am reminded that we probably should be normalizing args.build_file, args.deps_file, and args.pieces_file to their os.path.joined and fully expanded versions in argument parsing rather than at use.  That's captured in this bug :https://github.com/ansible-community/antsibull/issues/19  No need to address that in this PR.  Just mentioning it so we can all see the eventual plan for this code.

Reply:
Rethinking this and talked with felixfontein on IRC.  The shell should have expanded tildes and variables.  So this code is fine and args.dest_dir has a bug.  I'll open a ticket to fix args.dest_dir.